### ISSUE TITLE
v1.4.4 missing out_newrelic.so file

### ISSUE THREAD

**Issue by bpschmitt** [newrelic/newrelic-fluent-bit-output#62]
When installing the latest helm chart, I see the following error in the Pod logs.  

```
k logs newrelic-logging-l6k5v -n newrelic
[proxy] error opening plugin /fluent-bit/bin/out_newrelic.so: '/fluent-bit/bin/out_newrelic.so: cannot open shared object file: No such file or directory'
[2021/01/10 23:30:50] [error] [plugin] error loading proxy plugin: /fluent-bit/bin/out_newrelic.so
```

Reverting back to tag `v1.4.3` allows me to install successfully.  Please review `v1.4.4`.

**Comment by bpschmitt** [comment]
Output from `kubectl describe pod` confirming the version pulled for the pods throwing errors.

```
Events:
  Type     Reason     Age                     From               Message
  ----     ------     ----                    ----               -------
  Normal   Scheduled  4m50s                   default-scheduler  Successfully assigned newrelic/newrelic-logging-l6k5v to ip-192-168-55-255.us-west-2.compute.internal
  Normal   Pulled     3m24s (x5 over 4m49s)   kubelet            Container image "newrelic/newrelic-fluentbit-output:1.4.4" already present on machine
  Normal   Created    3m24s (x5 over 4m49s)   kubelet            Created container newrelic-logging
  Normal   Started    3m24s (x5 over 4m49s)   kubelet            Started container newrelic-logging
  Warning  BackOff    2m54s (x10 over 4m47s)  kubelet            Back-off restarting failed container
  ```

**Comment by bpschmitt** [comment]
Output from `kubectl describe pod` after reverting back to `v.1.4.3`.

```
Events:
  Type    Reason     Age   From               Message
  ----    ------     ----  ----               -------
  Normal  Scheduled  10s   default-scheduler  Successfully assigned newrelic/newrelic-logging-srczq to ip-192-168-55-255.us-west-2.compute.internal
  Normal  Pulled     10s   kubelet            Container image "newrelic/newrelic-fluentbit-output:1.4.3" already present on machine
  Normal  Created    10s   kubelet            Created container newrelic-logging
  Normal  Started    9s    kubelet            Started container newrelic-logging
  ```

**Comment by jsubirat** [comment]
Hi @bpschmitt , the file is now actually named `out_newrelic-linux-amd64-1.4.4.so` and is included in the Docker image. We just need to update the Helm charts with the new launch command. This is not a plugin issue then, so I'm closing this issue.

**Comment by jsubirat** [comment]
Hi again @bpschmitt . Actually the Docker file is incorrect, as reported in https://github.com/newrelic/newrelic-fluent-bit-output/issues/60. I'll fix it in v1.4.5, so please just use this version in your PR for the Helm chart.
